storage/data-plane/pom.xml

```diff
@@ -62,52 +45,58 @@
     </pluginRepositories>
 
     <dependencies>
+
         <dependency>
             <groupId>com.microsoft.rest.v2</groupId>
             <artifactId>client-runtime</artifactId>
-            <version>2.0.2</version>
         </dependency>
+
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>1.7.25</version>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
         </dependency>
+
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- Adding latest Netty as a temporary override of netty lib in v2 runtime.
+        The old version prevents tests from running in Java 11. Issue is raised to have netty updated in v2 runtime
+         Issue Link: https://github.com/Azure/azure-sdk-for-java/issues/3107 -->
         <dependency>
-            <groupId>org.spockframework</groupId>
-            <artifactId>spock-core</artifactId>
-            <scope>test</scope>
-            <version>1.1-groovy-2.4</version>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-all</artifactId>
+          <version>${netty-all.version}</version>
```

Comment:
Can you confirm that these versions need to be specified here and can not be inherited from the parent pom?

Reply:
Originally, I added these to ensure storage code using v2 runtime compiles in Java 11.
if we end up using these in for Storage v11  to compile with java 11.
The ideal place would be v2 runtime to add these libraries to.
I did open an issue to update v2 runtime with these netty libraries: 
Azure/azure-sdk-for-java#3107
If we end up needing them, will open a PR on v2 runtime.